### PR TITLE
Feature add n8n.io

### PR DESCRIPTION
**Pull request by LiquidITGuy**
few words about this MR : 
- https does not work (even with properly configured docker image (the clients works, but the api seems unreachable)
- the service uses basic web auth to autenticate user and it is removable. I tried to not use the auth from n8n but the one from caprover.
n8n service provides api creation for webhook that are secured from other way, activate caprover securisation will block the webhook function.
Second reason : if we activate the classical authent from n8n, with 2 web auth, we got some problems.

I would love to add notes to explain it more but there is no place so I put it all into the end part.
ps : to try trigger, dont forget to start the trigger listening on test or activate in production

- [X] I have tested the template using the method described in README.md thoroughly
- [X] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [X] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [X] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [X] Icon is added as a png file to the logos directory.
- [X] Documentation field contains a link to a page with proper explanations on environmental variables, volumes or the docker compose file.

### CODE REVIEW

**Comment by LiquidITGuy** [comment]
2nd précision : it will work in https but with an error (connexion lost) but it seems to work very well (issue should be created on n8n side)

**Comment by LiquidITGuy** [comment]
Capture d’écran 2020-05-24 à 01 09 07
Capture d’écran 2020-05-24 à 01 09 38
on http:
Capture d’écran 2020-05-24 à 01 09 51
on https : 
Capture d’écran 2020-05-24 à 01 10 06

File: public/v2/apps/n8n-io.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      "description": "Timezone used by the server. Please check out this page for valid time"
+    },{
+      "id": "$$cap_user",
+      "label": "user",
+      "defaultValue": "root",
+      "description": "user you need to use to login into the web interface"
+    },{
+      "id": "$$cap_pass",
+      "label": "password",
+      "description": "password you need to use to login into the web interface"
```
Comment (githubsaturn) [inline public/v2/apps/n8n-io.json:91]: valid regex needs to be added to all params here.

Comment (LiquidITGuy) [reply public/v2/apps/n8n-io.json]: done

**Pull request merged**